app/graphql/types/group_filter_type.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Types
+  # Group Filter Type
+  class GroupFilterType < BaseInputObject # rubocop:disable GraphQL/ObjectDescription
+    Group.ransackable_attributes.excluding('id').each do |attr|
```

Comment:
**nitpick (blocking):** Can you exclude `deleted_at` as well since we currently don't allow querying of deleted data through the api.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/749/commits/e614cc463c50c7d65583d1bfdb60134387e31c61